src/print-prescription/prescription-printout.component.tsx

```diff
@@ -0,0 +1,119 @@
+  medicationrequests: Array<MedicationRequestBundle>;
+  excludedPrescription: Array<string>;
+};
+
+const PrescriptionsPrintout: React.FC<PrescriptionsPrintoutProps> = ({ medicationrequests, excludedPrescription }) => {
+  const { t } = useTranslation();
+  const {
+    sessionLocation: { display: facilityName },
+  } = useSession();
+  const patient = medicationrequests[0]?.request?.subject;
+  const extractpatientName = (display: string) => display.match(/^([A-Za-z\s]+)\s\(/)?.at(1);
+  return (
+    <Layer className={styles.printOutContainer}>
+      <StructuredListWrapper>
+        <StructuredListBody>
+          <StructuredListRow head>
+            <StructuredListCell head>
+              <br />
+              <br />
+              <p className={styles.printoutTitle}>{t('prescriptionInstructions', 'Prescription Instructions')}</p>
+              {patient && (
+                <p className={styles.faintText} style={{ textAlign: 'center' }}>
+                  {extractpatientName(patient.display)?.toUpperCase()}
+                </p>
+              )}
+              <br />
+              <br />
+            </StructuredListCell>
+          </StructuredListRow>
+          {medicationrequests
+            ?.filter((req) => !excludedPrescription.includes(req.request.id))
+            ?.map((request, index) => {
+              const medicationEvent = request.request;
+              const dosageInstruction: DosageInstruction = getDosageInstruction(medicationEvent.dosageInstruction);
+              const quantity: Quantity = getQuantity(medicationEvent);
+              const refillsAllowed: number = getRefillsAllowed(medicationEvent);
+              return (
+                <div key={index}>
+                  {dosageInstruction && (
+                    <StructuredListRow>
+                      <StructuredListCell>
+                        <p>
+                          <strong>
+                            {getMedicationDisplay(getMedicationReferenceOrCodeableConcept(medicationEvent))}
+                          </strong>
+                        </p>
+                        <br />
+                        <p>
+                          <span className={styles.faintText}>{t('dose', 'Dose')}</span>
+                          {': '}
+                          <span>
+                            {dosageInstruction?.doseAndRate?.map((doseAndRate, index) => {
+                              return (
+                                <span key={index}>
+                                  {doseAndRate?.doseQuantity?.value} {doseAndRate?.doseQuantity?.unit}
+                                </span>
+                              );
+                            })}
+                          </span>{' '}
+                          &mdash; {dosageInstruction?.route?.text} &mdash; {dosageInstruction?.timing?.code?.text}{' '}
+                          {dosageInstruction?.timing?.repeat?.duration
+                            ? 'for ' +
```

Comment:
Any display text _must_ be translatable.

Reply:
fixed, thanks